ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__lp-interop-ocp4.12.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
I've seen later versions of golang be used here what do you think about using a newer version?

I don't currently have a reason to use a newer one, all versions I've tried here seem to work fine, just curious to get your thoughts.

Reply:
I don't mind giving it a shot, will submit in a commit soon.

Reply:
Added this in [3fd91bd](https://github.com/openshift/release/pull/36181/commits/3fd91bd72531fc6e504ff9eb26657e0278dea6d1). Will test today.

Reply:
Seems to work with 1.19